TrafficCapture/dockerSolution/src/main/docker/docker-compose.yml

```diff
@@ -43,9 +43,7 @@ services:
         condition: service_started
       opensearchtarget:
         condition: service_started
-      trafficcomparator:
-        condition: service_healthy
-    command: /bin/sh -c "/runJavaWithClasspath.sh org.opensearch.migrations.replay.TrafficReplayer https://opensearchtarget:9200 --auth-header-value Basic\\ YWRtaW46YWRtaW4= --insecure --kafka-traffic-brokers kafka:9092 --kafka-traffic-topic logging-traffic-topic --kafka-traffic-group-id default-logging-group | nc trafficcomparator 9220"
+    command: /bin/sh -c "/runJavaWithClasspath.sh org.opensearch.migrations.replay.TrafficReplayer https://opensearchtarget:9200 --auth-header-value Basic\\ YWRtaW46YWRtaW4= --insecure --kafka-traffic-brokers kafka:9092 --kafka-traffic-topic logging-traffic-topic --kafka-traffic-group-id default-logging-group"
```

Comment:
We still need to pipe the output here _somewhere_ or all the tuples will end up in our logs. So something like `| /dev/null` (and probably create a ticket to stop writing the tuples to stdout

Reply:
This is what we do within the TrafficReplayer today to write tuples both to a logfile and to output, either specified by a command line argument OR stdout...
```
        @Override
        @SneakyThrows
        public void accept(SourceTargetCaptureTuple triple) {
            JSONObject jsonObject = toJSONObject(triple);

            tupleLogger.info(()->jsonObject.toString());
            outputStream.write((jsonObject.toString()+"\n").getBytes(StandardCharsets.UTF_8));
            outputStream.flush();
        }
```

I don't think that we'll EVER want to use stdout to pipe data to another process.  tupleLogger will write the data as json to rotating logfiles.  I'd like to see outputStream get killed here & all the way up through the TrafficReplayer to remove the argument.  Notice that we write the string twice.  It may not make sense to buffer it, given that some of these are 10MB, we may not get great caching effects.  Removing a write that we don't care about will be a major win though.

As far as value in tuples, I've generally disabled these from my console logs as they easily dominate the output and drown out everything else.